src/convertToObject.js

```diff
@@ -6,7 +6,22 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
-}
+  const stylesArray = sourceString
+  .trim()
+  .split(';')
+  .map(style => style.trim())
+  .filter(style => style);
```

Comment:
The chaining of methods `.trim()`, `.split(';')`, `.map()` and `.filter()` should have the same indentation to improve readability. Align them under each other to follow good coding practices.